wagtail/admin/static_src/wagtailadmin/scss/components/_header.scss

```diff
@@ -50,6 +50,10 @@ header {
         margin-bottom: 0;
     }
 
+    &.tab-merged {
+        overflow-x: hidden;
+    }
```

Comment:
Ah, this is a whole story in itself… This is to prevent the streamfield for overflowing when the summary of a collapsed block is too large for the page. This `overflow-x` doesn’t cut the streamfield, but it allows to correctly set the max width of all descendants, including React streamfields. It took me more than 2 days to narrow it down. The whole Wagtail admin structure should be rewritten with flex boxes to fix this whole mess. Because this hack is just a little fix of a lot more pre-existing hacks.